src/main/java/com/virtuslab/using_directives/custom/Scanner.java

```diff
@@ -412,10 +419,14 @@ private void fetchToken() {
     td.lineOffset = td.lastOffset < reader.lineStartOffset ? reader.lineStartOffset : -1;
     td.name = null;
     char ch = reader.ch;
-    if (ch == ' ' || ch == '\t' || ch == CR || ch == LF || ch == FF) {
+    while (ch == ' ' || ch == '\t' || ch == CR || ch == LF || ch == FF) {
```

Comment:
do while maybe?

Reply:
Good idea, had to work around `reader.nextChar()`, but looks better regardless.